xtext-website/index.html

```diff
@@ -252,28 +252,28 @@ <h2>Editor Features By Platform</h2>
       <tr class="has-popover"
       	data-content="If your language interoperates with Java, the source files are considered during debug sessions in your IDE, i.e. you can set breakpoints and step through statements.">
         <td>Debugging</td>
-        <td><div class="supported"></div></td>
+        <td><div class="not-supported"></div></td>
         <td><div class="supported"></div></td>
         <td><div class="not-supported"></div></td>
       </tr>
       <tr class="has-popover"
       	data-content="Disable the selected lines of code by inserting comments into each line, or reenable them by removing such comments.">
         <td>Toggle Comments</td>
-        <td><div class="supported"></div></td>
+        <td><div class="not-supported"></div></td>
         <td><div class="supported"></div></td>
         <td><div class="not-supported"></div></td>
       </tr>
       <tr class="has-popover"
       	data-content="Display an outline of a file highlighting its structure. This is useful for quick navigation.">
         <td>Outline / Structure View</td>
-        <td><div class="supported"></div></td>
+        <td><div class="not-supported"></div></td>
```

Comment:
there is local symbols, which is interpreted to a hierarchical outline by some editors.